src/DssSpell.sol

```diff
@@ -52,18 +40,7 @@ contract SpellAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
 
-    // No rate chagnes this week.
-
-    function execute() external {
-        // Set the surplus buffer to 4,000,000 DAI
-        VowAbstract(MCD_VOW).file("hump", 4 * MILLION * RAD);
-
-        // Whitelist B Protocol to access the ETHUSD Medianizer
-        MedianAbstract(ETHUSD_MEDIAN).kiss(B_PROTOCOL_BUD_CONNECTOR);
-
-        // Whitelist B Protocol to access the ETHUSD OSM
-        OsmAbstract(ETHUSD_OSM).kiss(B_PROTOCOL_BUD_CONNECTOR);
-    }
+    function execute() external { }
```

Comment:
Not sure if it's worthwhile, but the last time we had one of these info-only spells we stuck a `require` check inside of the execute.

See:
https://github.com/makerdao/spells-mainnet/blob/master/archive/2020-05-25-DssSpell.sol#L30

Reply:
Good catch, we do typically do this.  This has the added benefit of making it real clear that governance has the authority at this moment in time.

Reply:
Yeah I like this, I'll approve once this is added.